lib/requirejs/require.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Wouldn't it be better to use a non-minified version of Require.JS for development purposes? Couldn't we use an npm version https://www.npmjs.com/package/requirejs?